test/mlir/driver/check_passthrough_options.mlir

```diff
@@ -1,13 +1,13 @@
-// RUN: onnx-mlir -Xopt --data-sections -v %s 2>&1 | FileCheck --check-prefix=OPT %s
-// RUN: onnx-mlir -Xllc --data-sections -v %s 2>&1 | FileCheck --check-prefix=LLC %s
-// RUN: onnx-mlir -mllvm --data-sections -v %s 2>&1 | FileCheck --check-prefix=LLVM %s
+// RUN: onnx-mlir -Xopt --data-sections -v %s -o %T/check_passthrough_options 2>&1 | FileCheck --check-prefix=OPT %s
```

Comment:
The lit manual says %T is deprecated?

Reply:
It does say so, but it is still used in llvm. I updated to the new recommendation anyway to make sure the tests aren't impacted in the future.